js-exercises/sum-fibs/sumFibs.js

```diff
@@ -0,0 +1,59 @@
+const isNumber = (arg) => typeof arg === 'number';
+
+const isOddNumber = (arg) => isNumber(arg) && (arg % 2 !== 0);
```

Comment:
This would be true for `NaN`

Reply:
right, fixed issue with bcb3526112c874b7365fbbf8a957a7479e2f727c